py-polars/polars/_utils/udfs.py

```diff
@@ -268,6 +268,7 @@ class OpNames:
     "json.loads": "str.json_decode",
 }
 _RE_IMPLICIT_BOOL = re.compile(r'pl\.col\("([^"]*)"\) & pl\.col\("\1"\)\.(.+)')
+_RE_STRIP_BOOL = re.compile(r"^bool\((.+)\)$")
```

Comment:
Now that we are seeing failing import tests, could we put all these globals behind a LRU cached function?

Reply:
We could, but they aren't relevant to the import timings (this module isn't imported until needed). I posted a detailed import timings breakdown in the dev chat and they haven't actually changed - something about CI env stability seems like a more likely culprit there 😢

Reply:
> this module isn't imported until needed

Ok, forget what I said. :smiley_cat: